woocommerce/woocommerce-gutenberg-products-block
# Start using the `block.json` metadata file for the `Featured Product` block

The goal of this PR is to use the `block.json` metadata file to register the `Featured Product` block.

Fixes https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/6239

### Testing

How to test the changes in this Pull Request:
1. Create a page and add a `Featured Product` block
2. Save the page and check it renders correctly the default config in the frontend
3. Edit the page and make some changes to the block (duotone, padding, overlay, focal point, etc.).
4. Save the page and check it renders with the new config.

### Changelog

> Start using the block.json file to register the Featured Product block
